.github/workflows/sphinx.yml

```diff
@@ -8,6 +8,7 @@ on:
   push:
     branches:
       - master
+      - main
```

Comment:
If we ever rename to main, nothing should actually break.

Reply:
Well, almost, there are some other places, where it's gonna break, I will address it after this PR.